Audible-Altimeter-Embedded/src/i2s.pio

```diff
@@ -0,0 +1,64 @@
+;
+; Copyright (c) 2020 Raspberry Pi (Trading) Ltd.
+;
+; SPDX-License-Identifier: BSD-3-Clause
+;
+
+.program pio_i2s
+.side_set 2
+; Just serialise a stream of bits. Take 32 bits from each FIFO record. LSB-first.
+                    ;        /--- LRCLK
+                    ;        |/-- BCLK
+bitloop1:           ;        ||
+    out pins, 1       side 0b10
+    jmp x-- bitloop1  side 0b11
+    out pins, 1       side 0b00
+    set x, 14         side 0b01
```

Comment:
When converting to 8 bit, `set x, 14` 

1. make sure audio is in mono
2. remove one of the bitloops
3. `set x, 6` instead of 14. This is the counter to make sure the. 2 less than 8 for 8 bits.

We should only have to use one bitloop IF it is mono.